tex/measure/lebesgue-int.tex

```diff
@@ -80,10 +87,25 @@ \section{The definition}
 	The constant function $1$ is \emph{not} integrable on $\RR$,
 	since $\int_\RR 1 \; d\mu = \mu(\RR) = +\infty$.
 \end{example}
-For this reason, I will usually prefer the term ``integrable''.
+For this reason, I will usually prefer the term ``absolutely integrable''.
 (If it were up to me, I would call it ``finitely integrable'',
 and usually do so privately.)
 
+\begin{remark}
+	[Why don't we approximate the integral from above?]
+	For bounded functions on measure spaces with $|\Omega| < \infty$, we can equivalently define
+	\[ \int_\Omega f \; d\mu \defeq
+		\inf_{0 \le s \le f} \left( \int_\Omega s \; d\mu \right) \]
+	where the infimum is taken over all simple $s$ such that $f \leq s$.
+	However, if the functions are unbounded or $|\Omega| = \infty$, the situation is not that simple:
+	\begin{itemize}
+		\ii Show that $f(x) = x^{-2}$ over $\Omega = (1, \infty)$ is absolutely integrable,
+		yet for all simple $s$ such that $f \leq s$, then $\int_\Omega f \; d\mu = \infty$.
+		\ii Show that $f(x) = x^{-0.5}$ over $\Omega = (0, 1)$ is absolutely integrable,
+		yet there's no simple $s$ such that $f \leq s$ and $s$ is finite almost everywhere.
+	\end{itemize}
+\end{remark}
```

Comment:
maybe this should be "exercise" and not "remark" if the imperative is "show that"